catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -75,11 +81,17 @@ define([
         validate: function() {
             var error;
             var indices = [];
+            var type = this.get('type');
+            if(!this.get('value') && (type === 'point' || type === 'bounds')) {
+                this.set('value',[]);
+            }
+
             if(!this.get('value') && !this.get('regex') && !this.get('required')){
                 if(this.hadError){
                     this.hadError = undefined;
                     wreqr.vent.trigger('fieldErrorChange:'+this.get('key'));
                 }
+                wreqr.vent.trigger('fieldChanged:'+this.get('parentId'));
```

Comment:
You added this wreqr event, but you didn't add anything that is listening to it. Also, having wreqr in a model is a code smell.